src/lib/db/feature-toggle-client-store.ts

```diff
@@ -300,13 +317,15 @@ export default class FeatureToggleClientStore
         featureQuery?: IFeatureToggleQuery,
         includeStrategyIds?: boolean,
         includeDisabledStrategies?: boolean,
+        includeStrategyTitles?: boolean,
```

Comment:
what's the story behind those include special cases? Are they for playground only while SDKs never include anythig?

Reply:
I don't know about the disabled strategies (but @andreas-unleash might?). The strategy IDs one **is** for playground, though. Don't know if it's been used anywhere else since, but probably not. Might be beneficial to abstract it away into a `getPlaygroundClient` function instead?

Reply:
BTW from the API design perspective includeXYZ: boolean is not very elegant. Maybe sth like:
` type OptionalInclude = 'strategyIds' | 'disabledStratgies' | 'strategyTitles' `
+ array of OptionalIncludes